### ISSUE TITLE
4.7.10 Cannot read property 'fontFamily' of undefined

### ISSUE THREAD

**Issue by stefanholzapfel** [tinymce/tinymce#4330]
**Do you want to request a *feature* or report a *bug*?**
Bug?

**What is the current behavior?**
After upgrade from 4.7.9 to 4.7.10, I get "TypeError: Cannot read property 'fontFamily' of undefined"  
![1](https://user-images.githubusercontent.com/9963536/38469778-5dc241ec-3b5a-11e8-832c-b66253b4b1f7.PNG)

If I revert, everything is fine.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**
My init config:
```
        tinymce.init({
            target: this.elementRef.nativeElement,
            skin_url: '../assets/tinymce/skins/lightgray',
            language: this.language,
            language_url: '../assets/tinymce/langs/' + this.language + '.js',
            inline: true,
            theme: 'modern',
            menubar: false,
            statusbar: false,
            toolbar_items_size: 'small',
            forced_root_block: 'p',
            fixed_toolbar_container: this.attachToolbarToElementWithId,
            toolbar: 'bold italic underline | ' +
            'forecolor backcolor | ' +
            'fontselect | ' +
            'fontsizeselect | ' +
            'alignleft aligncenter alignright alignjustify | ' +
            'removeformat',
            font_formats: 'Arial=arial,helvetica,sans-serif;' +
            'Courier New=courier new,courier,monospace;' +
            'Helvetica=helvetica;' +
            'Tahoma=tahoma,geneva,sans-serif;' +
            'Times New Roman=times new roman,times;' +
            'Verdana=verdana,geneva;',
            fontsize_formats: '0.25em 0.5em 0.75em 1em 1.25em 1.5em 1.75em 2em 2.5em 3em 3.5em 4em',
            plugins: 'paste textcolor colorpicker',
            paste_as_text: true,
            setup: editor => {
                this.editor = editor;
                editor.on('ExecCommand', (event) => {
                    this.dispatchInputEvent();
                });
                editor.on('KeyUp', (event) => {
                    this.keyUp.emit(event);
                    // Fire input event manually on enter, backspace or delete, since they don't trigger change
                    // when used on empty lines
                    if (event.keyCode === ENTER || event.keyCode === BACKSPACE || event.keyCode === DELETE) {
                        this.dispatchInputEvent();
                    }
                });
                editor.on('KeyDown', (event) => {
                    this.keyDown.emit(event);
                });
            }
        }).then((editorInstance) => {
            this.editor = editorInstance[0];
        });
```

**What is the expected behavior?**
No error :)

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Tested in Chrome 65.0.3325.181

**Comment by fyrkant** [comment]
Looks like this is being used in a Angular application, have you tried using our [official Angular wrapper](https://github.com/tinymce/tinymce-angular)?

**Comment by fyrkant** [comment]
This has been fixed in the newly released 4.7.11 release.
